docs/DeveloperGuide.md

```diff
@@ -754,18 +967,147 @@ testers are expected to do more *exploratory* testing.
+       Expected: Similar to previous.
+
+### Editing an applicant
+
+1. Editing an applicant while all applicants are being shown
+
+    1. Prerequisites: List all applicants using the `list` command. Multiple applicants in the list.
+
+    1. Test case: `edit 1 n/Tom Greenwood`<br>
+       Expected: The name of the first applicant is updated to *Tom Greenwood*. Updated details of the applicant shown in the response area.
+       Working area shows the updated list of applicants.
+
+    1. Test case: `edit n/Pop Greenwood`<br>
+       Expected: No applicant is edited. Error details shown in the response area. Applicant list in working area remains the same.
+
+    1. Other incorrect edit commands to try: `edit`, `edit x n/Jane Doe`, `edit e/email` (where x is larger than the list size)<br>
+       Expected: Similar to previous.
+
 ### Deleting an applicant
 
 1. Deleting an applicant while all applicants are being shown
 
     1. Prerequisites: List all applicants using the `list` command. Multiple applicants in the list.
 
     1. Test case: `delete 1`<br>
-       Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message.
-       Timestamp in the status bar is updated.
+       Expected: First applicant is deleted from the list. Details of the deleted applicant shown in the response area.
+       Working area shows the updated list of applicants.
 
     1. Test case: `delete 0`<br>
-       Expected: No applicant is deleted. Error details shown in the status message. Status bar remains the same.
+       Expected: No applicant is deleted. Error details shown in the response area. Applicant list in working area remains the same.
+
+    1. Other incorrect delete commands to try: `delete`, `delete x`, `delete a` (where x is larger than the list size)<br>
+       Expected: Similar to previous.
+
+### Editing an applicant's status
+
+1. Editing an applicant's status while all applicants are being shown
+
+    1. Prerequisites: List all applicants using the `list` command. Multiple applicants in the list.
+
+    1. Test case: `status 1 s/o`<br>
+       Expected: The status of the first applicant is updated to *OFFERED*. Updated details of the applicant shown in the response area.
+       Working area shows the updated list of applicants.
+
+    1. Test case: `status 1 s/l`<br>
+       Expected: No applicant's status is edited. Error details shown in the response area. Applicant list in working area remains the same.
+
+    1. Other incorrect edit status commands to try: `status`, `status x s/o`, `status 1 s/` (where x is larger than the list size)<br>
+       Expected: Similar to previous.
+
+### Adding an interview to an applicant
+
+1. Adding an interview to an applicant while all applicants are being shown
+
+    1. Prerequisites: List all applicants using the `list` command. Multiple applicants in the list.
+
+    1. Test case: `addi 1 t/technical r/8.6`<br>
```

Comment:
maybe add one more test case on the decimal places of the interview? or the length of the interview name

Reply:
Thanks for the review jingting! Will make the necessary changes 😄